CONTRIBUTING.md

```diff
@@ -36,6 +36,8 @@ To send us a pull request, please follow these steps:
 5. Send us a pull request with a [conventional semantic title](https://github.com/awslabs/aws-lambda-powertools-python/pull/67), and answering any default questions in the pull request interface.
 6. Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
 
+Optionally you can use [Gitpod](https://gitpod.io/) to have a pre-configured dev environment. If you choose to use Gitpod, there is no need to install anything on your machine, as the environment will be pre-configured with all necessary dependencies.  
```

Comment:
Hello @heitorlessa and @rubenfonseca. I'm really not sure if this information is enough to make it clear how to use Gitpod.io or we should create a section to explain how Gitpod works and how to use it. I need help :pray:

Reply:
Hi Leandro! This is great, but I'm not familiar with gitpod, and after reading this paragraph I don't know what I have to do exactly. Do I need to click somewhere? Do I have to go to gitpod.io instead? Some extra instructions would help a lot :)

Reply:
Hello Rubens! Currently we need to fork the repository, install all dependencies locally and start coding, basically using Gitpod we just need to fork the repository and click on that [badge](https://github.com/awslabs/aws-lambda-powertools-python/pull/1403/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R6) and everything is ready to start coding.
To contribute to the project this document guides us through 7 steps and using Gitpod basically changes step 2 because we no longer need to install dependencies locally.

What do you think about this:

`Optionally you can use [Gitpod](https://gitpod.io/) to have a pre-configured dev environment. If you choose to use Gitpod, there is no need to install anything on your machine, as the environment will be pre-configured with all necessary dependencies. To gain access to a pre-configured environment using Gitpod, simply fork the repository and click the Gitpod badge in the README.md file.`

Reply:
I'd suggest updating the entire section and use something along these lines. This makes our recommendation to use our pre-configured env explicit, give them recommendations on how to create against their fork (not ours, or else their push will fail), and keeping the remaining steps applicable for both.

The only thing I kinda miss is having a link for people unfamiliar with GitHub trying to create their first PR - maybe installing GitHub CLI and adding [this link](https://cli.github.com/manual/examples.html) can raise the bar? I'm fine without it for now too.

---

Firstly, [fork the repository](https://github.com/awslabs/aws-lambda-powertools-python/fork).

To setup your development environment, we recommend using our pre-configured Cloud environment: https://gitpod.io/#https://github.com/YOUR_USERNAME/aws-lambda-powertools-python. Replace `YOUR_USERNAME` with your GitHub username so the Cloud environment can target your fork accordingly.

Alternatively, you can use `make dev` within your local virtual environment.

To send us a pull request, please follow these steps:

1. Create a new branch to focus on the specific change you are contributing e.g. `improv/logger-debug-sampling`
2. Run all tests, and code baseline checks: `make pr`
    - Git hooks will run linting and formatting while `make pr` run deep checks that also run in the CI process
3. Commit to your fork using clear commit messages.
4. Send us a pull request with a [conventional semantic title](https://github.com/awslabs/aws-lambda-powertools-python/pull/67), and answering any default questions in the pull request interface.
5. Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.